### PR TITLE
Flatbuffers ns11

### PR DESCRIPTION
**Pull request by umithardal**
This PR introduces FlatBuffers `ns11` schema.

@mattclarke I pushed the code as a draft so you can monitor what I am doing and interfere if necessary as I am not perfectly comfortable with FlatBuffers. 

- [x] Finish serialise and deserialise methods for ns11
- [x] Add unit test and update `init`
- [ ] C++ verification and update README

### CODE REVIEW

File: streaming_data_types/nicos_typed_cache_ns11.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import flatbuffers
+from streaming_data_types.\
+    fbschemas.nicos_typed_cache_ns11 import TypedCacheEntry
+from streaming_data_types.utils import check_schema_identifier
+
+
+FILE_IDENTIFIER = b"ns11"
+
+
+def serialise_ns10(key: str, value: str, time_stamp: float = 0,
```
Comment (mattclarke) [inline streaming_data_types/nicos_typed_cache_ns11.py]:
```suggestion
def serialise_ns11(key: str, value: str, time_stamp: float = 0,
```

File: streaming_data_types/nicos_typed_cache_ns11.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    builder.Finish(cache_entry_message)
+
+    # Generate the output and replace the file_identifier
+    buffer = builder.Output()
+    buffer[4:8] = FILE_IDENTIFIER
+
+    return bytes(buffer)
+
+
+def deserialise_ns10(buffer):
```
Comment (mattclarke) [inline streaming_data_types/nicos_typed_cache_ns11.py]:
```suggestion
def deserialise_ns11(buffer):
```

Comment (umithardal) [reply streaming_data_types/nicos_typed_cache_ns11.py]: just did :)

**Comment by umithardal** [comment]
I have implemented ns11 for 4 basic types (string, float, int and bool). The current general test succesfully passes. Now, I will implement the remaining types and separate tests for each type.

**Comment by umithardal** [comment]
Note that I will make tests parametrised after I add deserialisation method.

**Pull request closed without merging**